_includes/layout/base/helper-step-2.html

```diff
@@ -0,0 +1,64 @@
+{% comment %}
+This file is licensed under the MIT License (MIT) available on
+http://opensource.org/licenses/MIT.
+{% endcomment %}
+
+<div class="helper-step" data-step="user">
+  <div class="helper-row">
+    <div class="helper-page-numbers">
+      <div class="helper-page-number completed"></div>
+      <div class="helper-page-number active">2</div>
+      <div class="helper-page-number">3</div>
+      <div class="helper-page-number">4</div>
+    </div>
+  
+    <div class="helper-content">
+      <div class="helper-header">
+        <div class="helper-header-row">
+          <h2 class="helper-title previous">What’s your operating system?</h2> 
+          <div class="row">
+            <div class="helper-selected-filter">Desktop wallets / OS X</div>
+            <button class="helper-change-btn">Change</button>
```

Comment:
@alexcherman Just a reminder that we need to make sure these `Change` buttons function as intended as users progress through the wizard.

Reply:
Let's also make sure that the parameters are updating correctly to show the selected values:

![image](https://user-images.githubusercontent.com/1130872/60332217-a2facb80-9996-11e9-8669-6844af00ac8f.png)

Reply:
Yes, I'm still working on it

Reply:
Ok, thanks.